### DISCUSSION TITLE
Support Environment Tag on the auth0_tenant Resource

### DISCUSSION THREAD

**Discussion by krikork** [auth0/terraform-provider-auth0#1177] [General]
### Checklist

- [X] I agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).

### Describe the problem you'd like to have solved

I could control the Environment Tag on the auth0_tenant Resource.

### Describe the ideal solution

Environment Tag is a configuration on the tenant I can control from the Auth0 management web app. I should be able to control it from TF as well.

**Comment by sergiught** [comment]
Hey @krikork thanks for raising this, unfortunately this is not yet something we can add within the terraform provider as we are missing the respective Auth0 Management API endpoints to allow managing the env tag. We'll leave this issue open until this changes and we can add support for this.
